sentry-android-core/src/main/java/io/sentry/android/core/InternalSentrySdk.java

```diff
@@ -184,6 +192,10 @@ public static SentryId captureEnvelope(final @NotNull byte[] envelopeData) {
       if (session != null) {
         final SentryEnvelopeItem sessionItem = SentryEnvelopeItem.fromSession(serializer, session);
         envelopeItems.add(sessionItem);
+        deleteCurrentSessionFile(options);
+        if (maybeStartNewSession) {
+          Sentry.startSession();
```

Comment:
Shouldn't make a real difference, but it's better to use hub here.
```suggestion
          hub.startSession();
```